src/components/ui/BaseLink.vue

```diff
@@ -4,7 +4,7 @@
 		class="link is-external"
 		:href="href"
 		v-bind="$attrs"
-		target="_blank"
+		:target="target"
```

Comment:
Maybe add it by default for urls that should use the fallback (all relative urls)? Then you don't need to specify `_self` on all the link components
```suggestion
		:target="href.startsWith('/') ? '_self' : target"
```